proposed/2023/package-source-mapping-status-PMUI.md

```diff
@@ -0,0 +1,78 @@
+# Package Source Mapping Status in the PM UI 
+* Start Date: 5/17/23
+* Authors: Donnie Goodson ([donnie-msft](https://github.com/donnie-msft))
+* Issue: https://github.com/NuGet/Home/issues/12586
+
+## Summary
+
+Currently, the NuGet Package Manager in Visual Studio (PM UI) does not support the package source mapping feature. Introducing source mapping status for the package selected in the Details Pane of PM UI is the first step to bringing this feature into the PM UI.
+
+Package Source Mapping in Restore was introduced in: https://github.com/NuGet/Home/blob/dev/implemented/2021/PackageSourceMapping.md
+
+## Motivation
+
+Showing in the PM UI Details Pane the package source mapping status, as well as a shortcut to manage the mappings in the VS Options dialog, will allow customers to onboard to or see the state of their package source mappings at a glance. The knowledge of whether a package is already mapped may lead a customer to modify their source mappings using the existing VS Options dialog. 
+
+As customers begin seeing source mapping status in the PM UI and becoming more aware of the feature, we can learn more about what they want from the PM UI as far as package management actions (i.e., installing/updating a package). The goal is to let customers drive the next evolution of package source mapping in PM UI.
+
+## Explanation
+
+### Functional Explanation
+
+On the details pane there will be new information shown below the Install/Update buttons. 
+
+#### Mapping Status
+
+A label will appear under the `Install/Update` button indicating the Package Source Mapping status.
+
+| Project Onboarded | Selected Package is Mapped | Status Icon | Status Text 
```

Comment:
Icons might be:

Disabled - Information or a Hyperlink type icon

Onboarded Not Mapped - X or a disconnected link

Onboarded Mapped - Checkmark or a connected link

Reply:
For not mapped / mapped, some that stick out...
`ConnectionOffline` is a little subtle. Maybe a use the `StatusOffline` and `ConnectionWarning`? I also like `StatusOK` for the good to go scenario.

1. x and check 
  ![image](https://github.com/NuGet/Home/assets/49205731/356a2abf-14f0-4bce-86e5-36c149720f4b)
  ![image](https://github.com/NuGet/Home/assets/49205731/1f891063-a373-441b-99be-be92f421950e)

1.  connection icons (
![image](https://github.com/NuGet/Home/assets/49205731/62e3fa00-69aa-40cc-b301-a550ae880fed)

1. Reuse nuget icons?
 ![image](https://github.com/NuGet/Home/assets/49205731/d2695fd4-b947-492a-8ce4-5f33feab565b)

Reply:
I think the icons here will end up being pretty small, so I wonder if icon options 2 and 3 would be too busy for this use case? I'd personally opt towards something more simple, like the icons in option 1

Reply:
Added icons to the table!